iTwin/viewer
# Initial Viewer CLI

To test:

1.  From a folder outside of your repo, execute "node path/to/viewer/packages/modules/create-itwin-viewer/bin
2. Follow the prompts

When you run the application, you will receive a runtime error about a dompurify import that should be resolved in [this PR ](https://github.com/iTwin/itwinjs-core/pull/3383)

Still need to implement the uiConfig, the desktop template, and some other items. Requesting comments in the meantime
